pyorientAssignment.py

```diff
@@ -36,14 +36,32 @@
 # then test each price to see if it is smaller than the current minimum. If it is, update 
 # the minimum variable with that price. You can do something similar to find the maximum.
 
+ini_min = 900  
+ini_max = 0
+sum_rec = 0
+
 for record in records:
 	print record.price
+	sum_rec += record.price
+	avg_price = sum_rec/len(records)
+	if record.price < ini_min:
+	    min_price = record.price
+	    ini_min = min_price
+	else:
```

Comment:
you do not need this else statement. If the current price is not lower than the minimum you simply go to the next price.